Formula/mutt.rb

```diff
@@ -11,9 +11,9 @@
 class Mutt < Formula
   desc "Mongrel of mail user agents (part elm, pine, mush, mh, etc.)"
   homepage "http://www.mutt.org/"
-  url "https://bitbucket.org/mutt/mutt/downloads/mutt-1.6.2.tar.gz"
-  mirror "ftp://ftp.mutt.org/pub/mutt/mutt-1.6.2.tar.gz"
-  sha256 "c5d02ef06486cdf04f9eeb9e9d7994890d8dfa7f47e7bfeb53a2a67da2ac1d8e"
+  url "ftp://ftp.mutt.org/pub/mutt/mutt-1.7.0.tar.gz"
```

Comment:
Would prefer to keep the Bitbucket mirror as the primary URL. `ftp` tends to get firewalled to death.